.github/labels.yaml

```diff
@@ -0,0 +1,808 @@
+  description: Item related to an archived level project or the archive criteria/process itself
+  color: d93f0b
+- name: level/graduation
+  description: Item related to a graduation level project or the graduation criteria/process itself
+  color: 0052cc
+- name: level/incubation
+  description: Item related to an incubation level project or the incubation criteria/process itself
+  color: ed0e81
+- name: level/sandbox
+  description: Item related to a sandbox level project or the sandbox criteria/process itself
+  color: e884e1
+- name: needs-group
+  description: Indicates an issue or PR that has not been assigned a group (toc or tag/foo label applied)
+  color: ededed
+- name: needs-kind
+  description: Indicates an issue or PR that is missing an issue type or kind (a kind/foo label)
+  color: ededed
+- name: needs-triage
+  description: Indicates an issue or PR that has not been triaged yet (has a 'triage/foo' label applied)
+  color: ededed
+- name: review/governance
+  description: Project Governance Review
+  color: '5319e7'
+- name: review/health
+  description: Project Health Review
+  color: '5319e7'
+- name: review/security
+  description: Project Security Review
+  color: '5319e7'
+- name: review/tech
+  description: Project Tech Review
+  color: '5319e7'
+- name: sub/contributor-strategy-and-advocacy
+  description: TOC Contributor Strategy and Advocacy SubProject
+  color: 924F23
+- name: sub/mentoring
+  description: TOC Mentoring Subproject
+  color: 6DCC2C
+- name: sub/project-reviews
+  description: TOC Project Review Subproject
+  color: '3E4469'
+- name: tag/developer-experience
+  description: TAG Developer Experience
+  color: c2e0c6
+- name: tag/infrastructure
+  description: TAG Infrastructure
+  color: D786DA
+- name: tag/operational-resilience
+  description: TAG Operational Resilience
+  color: 83075D
+- name: tag/security-and-compliance
+  description: TAG Security and Compliance
+  color: 78C971
+- name: tag/workloads-foundation
+  description: TAG Workloads Foundation
+  color: 6AE2DC
+- name: toc
+  description: toc specific issue
+  color: CF0CBE
+- name: toc/initiative/AI
+  description: TOC Artificial Intelligence Initiative
+  color: d4c5f9
+- name: triage/duplicate
+  description: Duplicate issue or PR, can be closed
+  color: ebf84a
+- name: triage/needs-information
+  description: Needs additional information provided before it can be worked on
+  color: b60205
+- name: triage/not-planned
+  description: Out of scope, or not planned to be worked on
+  color: d93f0b
+- name: triage/valid
+  description: Issue or PR is valid with enough information to be actionable
+  color: 0e8a16
+- name: vote open
+  description: ''
+  color: ededed
+- name: vote/closed
+  description: An election that has been completed
+  color: 5c2908
+- name: vote/nomination
+  description: A nomination or call for nominations
+  color: E00EEF
+- name: vote/open
+  description: An election is open
+  color: 0e8a16
```

Comment:
Is it possible that these labels may be confused with the TOC voting on projects for moving levels?  the description is clear, but the description isnt prominent via the label through the UI interface.

Reply:
I don't think so? they have to be manually applied, and someone would have to do that. We also use gitvote for those. These are for the other things such as leadership elections. 

If it'd help I can specify non moving levels related elections

Reply:
So while i understand what they are for (the description says it here) when reviewing issues (particularly when not looking at a single issue, but an overview of several) that have the label on them it may not be apparent what the vote is about without viewing the direct issue in question. 

nomination is clear. but vote/open and vote/closed are not clear. it is just that a vote is occurring or has occurred.

we can use these generally but if we're talking about elections, we may want to be explicit about it.